types/list.js/index.d.ts

```diff
@@ -60,4 +60,5 @@ declare class ListItem {
     visible(): boolean;
 }
 
-export = List;
+export default List;
```

Comment:
This module exports a plain function. In such a case you shouldn't replace the `export = ...` with `export default ...` statement. You can find more materials on this topic [here](https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

I suppose the reason for your change was not (only?) to replace the export, but to export the interfaces, too. You can achieve this keeping the `export = ...` syntax:
```typescript
declare class List {
  // ...
}

declare namespace List {
  // all interfaces go here
}

export = List;
```

You will have to change `ListItem` from `class` to `interface` for this to work.